tests/webapp/api/test_performance_data_api.py

```diff
@@ -683,6 +683,40 @@ def test_perfcompare_results_multiple_runs(
 
     option_collection_map = perf_compare_utils.get_option_collection_map()
 
+    first_row = dict()
+    first_row['base_avg_value'] = statistics.mean(sig1_values)
+    first_row['new_avg_value'] = statistics.mean(sig2_values)
+    first_row['is_improvement'] = perf_compare_utils.is_improvement(
```

Comment:
are you also going to add changes for `one_run`

Reply:
I've done the same thing for the other test as well.